app/Managers/SignupManager.php

```diff
@@ -44,11 +45,16 @@ public function create($data, $northstarId, $campaignId)
             $data['dont_send_to_blink']
         );
 
-        // Save the new signup in Customer.io, via Blink.
-        if (config('features.blink') && $shouldSendToCustomerIo) {
+        // Send the new signup to Customer.io:
+        if ($shouldSendToCustomerIo) {
             SendSignupToCustomerIo::dispatch($signup);
         }
 
+        // If this wasn't triggered via SMS, send to Gambit:
+        if (!preg_match('/(sms|gambit)/', $signup->source)) {
```

Comment:
This is to [replicate this logic in Blink](https://github.com/DoSomething/blink/blob/dfe6d5ef138bd2fb3d66a105d626f4ab6f62d6c5/src/workers/GambitCampaignSignupRelayWorker.js#L50-L56). And just in case we ever update this `source` to read the application name as we do for other sources, I figured we may as well include the string `gambit` in this check.